uberstyle/.golangci.yml

```
@@ -0,0 +1,250 @@
+---
+# golangci-lint configuration file made by @ccoVeille
+# Source: https://github.com/ccoVeille/golangci-lint-config-examples/
+# Author: @ccoVeille & @manuelarte
+# License: MIT
+# Variant: uberstyle
+# Version: v2.1.0
+#
+version: "2"
+
+formatters:
+  enable:
+    # format the code
+    - gofmt
+    # checks if the code and import statements are formatted according to the 'goimports' command.
+    - goimports
+
+  settings:
+    # format the code with Go standard library
+    gofmt:
+      # simplify the code
+      # https://pkg.go.dev/cmd/gofmt#hdr-The_simplify_command
+      simplify: true
+      rewrite-rules:
+        # replace `interface{}` with `any` in the code on format
+        - pattern: "interface{}"
+          replacement: "any"
+
+linters:
+  exclusions:
+    # these presets where present in the v1 version of golangci-lint
+    # it's interesting to keep them when migrating, but removing them should be the goal
+    presets:
+      # exclude check on comments format in godoc
+      # These are common false positives in poor code
+      # you should not use this on recent code you write from scratch
+      # More information: https://golangci-lint.run/usage/false-positives/#comments
+      #
+      # Please uncomment the following line if your code is using the godoc format
+      # - comments
```

My bad, it should be this

```suggestion
      # Please comment the following line if your code is using the godoc format
       - comments
```

---

done